cfpb/consumerfinance.gov
# Remove old feature-flagged banners

We've migrated all our old outage banners into our Wagtail banner manager, so we can get rid of all the infrastructure that supported them.

---

## Removals

- Banner feature flags
- Banner templates
- Banner template tags
- Banner unit tests

## How to test this PR

1. Pull down a copy of the latest production database
2. Check that all the banners removed here can be turned on from the Wagtail banner manager (/admin/v1/banner/), ignoring small changes we made to simplify the banners (i.e. removing the fancy automated date and time insertion in the complaint maintenance banner)
3. Check to make sure nothing else banner-related seems broken, I guess?

## Notes and todos

- The old banner stuff was scattered in a lot of places. Did I miss anything? Did I take away too much?
- TODO: Once these changes are deployed, we'll need to purge the removed feature flags from the database to get them to stop showing up in Wagtail

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
